fairtools/MCConfigurator/FairYamlVMCConfig.cxx

```diff
@@ -183,22 +190,25 @@ void FairYamlVMCConfig::SetupGeant4()
         //        LOG(info) << "Setting Geant4 multithreaded to " << (mtMode?"true":"false");
     }
 
-    FairFastSimRunConfiguration* runConfiguration =
-        new FairFastSimRunConfiguration(fYamlConfig["Geant4_UserGeometry"].as<std::string>(),
-                                        fYamlConfig["Geant4_PhysicsList"].as<std::string>(),
-                                        fYamlConfig["Geant4_SpecialProcess"].as<std::string>(),
-                                        specialStacking,
-                                        mtMode);
+    auto const useFastSim(fYamlConfig["UseFastSim"] ? fYamlConfig["UseFastSim"].as<bool>() : fUseFastSimDefault);
+    std::unique_ptr<TG4RunConfiguration> runConfiguration;
+    if (useFastSim) {
+        runConfiguration = std::make_unique<FairFastSimRunConfiguration>(
+            g4UserGeometry, g4PhysicsList, g4SpecialProcess, specialStacking, mtMode);
+    } else {
+        runConfiguration = std::make_unique<TG4RunConfiguration>(
+            g4UserGeometry, g4PhysicsList, g4SpecialProcess, specialStacking, mtMode);
+    }
```

Comment:
`std::make_unique` requires C++14. FairRoot currently is saying, that it needs C++11.

Please either bump to C++14 (fine with me, I would like to add some `[[deprecated]]`).

Or rewrite using `.reset(new …)`.

Reply:
Thx, I will address this!

Reply:
done